hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/StreamCommitWatcher.java

```diff
@@ -172,28 +172,17 @@ private void adjustBuffers(long commitIndex) {
     }
   }
 
-  private long releaseBuffers(List<Long> indexes) {
+  private void releaseBuffers(List<Long> indexes) {
     Preconditions.checkArgument(!commitIndexMap.isEmpty());
     for (long index : indexes) {
       Preconditions.checkState(commitIndexMap.containsKey(index));
       final List<StreamBuffer> buffers = commitIndexMap.remove(index);
-      final long length =
-          buffers.stream().mapToLong(StreamBuffer::position).sum();
-      totalAckDataLength += length;
```

Comment:
I don't think we can remove the totalAckDataLength computation  part because this is used during [retry](https://github.com/apache/ozone/blob/cd143c543780551e8cc77bcf201ba74125f90dea/hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/KeyDataStreamOutput.java#L281)

Reply:
Thanks for reminding. I had update the patch, and revert totalAckDataLength computation.